docs/.vitepress/config.ts

```diff
@@ -129,161 +114,695 @@ export default defineConfig({
+            { text: 'Terramate Scripts', link: '/cli/orchestration/scripts' },
+            { text: 'Tag Filter', link: '/cli/orchestration/tag-filter' },
+            { text: 'Runtime Configuration', link: '/cli/orchestration/runtime-configuration' },
+            { text: 'Safe Guards', link: '/cli/orchestration/safeguards' },
           ],
         },
         {
-          text: '🔗 Sharing Data',
+          text: '�️ Change Detection',
           collapsed: false,
           items: [
-            { text: 'Overview', link: '/cli/data-sharing/' },
-            { text: 'Globals', link: '/cli/data-sharing/globals' },
-            { text: 'Metadata', link: '/cli/data-sharing/metadata' },
-            { text: 'Map', link: '/cli/map' },
+            { text: 'Overview', link: '/cli/change-detection/' },
+            {
+              text: 'Integrations',
+              collapsed: true,
+              items: [
+                { text: 'Git Integration', link: '/cli/change-detection/integrations/git' },
+                { text: 'Terraform Integration', link: '/cli/change-detection/integrations/terraform' },
+                { text: 'OpenTofu Integration', link: '/cli/change-detection/integrations/opentofu' },
+                { text: 'Terragrunt Integration', link: '/cli/change-detection/integrations/terragrunt' },
+              ],
+            },
+            { text: 'File Watchers', link: '/cli/change-detection/file-watchers' },
           ],
         },
         {
-          text: '😍 Code Generation',
+          text: '🔗 Code Generation',
           collapsed: false,
           items: [
             { text: 'Overview', link: '/cli/code-generation/' },
             { text: 'Generate HCL', link: '/cli/code-generation/generate-hcl' },
             { text: 'Generate File', link: '/cli/code-generation/generate-file' },
-          ],
-        },
-        {
-          text: '🔧 Functions',
-          collapsed: false,
-          items: [
             {
-              text: 'Overview',
-              link: '/cli/functions/index',
+              text: 'Variables',
+              collapsed: true,
+              items: [
+                { text: 'Overview', link: '/cli/code-generation/variables/' },
+                { text: 'Globals', link: '/cli/code-generation/variables/globals' },
+                { text: 'Lets', link: '/cli/code-generation/variables/lets' },
+                { text: 'Metadata', link: '/cli/code-generation/variables/metadata' },
+                { text: 'Map', link: '/cli/code-generation/variables/map' },
+              ],
             },
             {
-              text: 'Terramate specific functions',
+              text: 'Functions',
+              collapsed: true,
               items: [
                 {
-                  text: 'tm_ternary',
-                  link: '/cli/functions/terramate-builtin/tm_ternary',
+                  text: 'Overview',
```

Comment:
On the `Commands (CLI)` [overview](https://deploy-preview-1306--docs-terramate-io.netlify.app/cli/cmdline/) page, the `Next Page` points to `Introduction` instead of pointing to the `Terramate Cloud login` page.
![image](https://github.com/terramate-io/terramate/assets/92498/4ba93368-d088-4fa4-83b6-693d5d660b51)

Reply:
Thanks! I just resolved this now

Reply:
Thanks! I just resolved this now